src/components/Input/CheckBox/CheckBoxInput.styles.js

```diff
@@ -15,8 +15,8 @@ export default styled.input.attrs({
   display: flex;
   justify-content: center;
   align-items: center;
-
-  &::-ms-check {
+  ${({ isTopAligned }) =>
```

Comment:
missing new line before `&::-ms-check`

Reply:
fixed those, btw `prettier` caused this removing of spaces. That's why I had to remove the func one line up.